health/notifications/health_alarm_notify.conf

```diff
@@ -763,6 +782,8 @@ custom_sender() {
 
 role_recipients_email[sysadmin]="${DEFAULT_RECIPIENT_EMAIL}"
 
+#role_recipients_hangouts[sysadmin]="${DEFAULT_RECIPIENT_HANGOUTS}"
```

Comment:
why this is commented-out?

Reply:
It was for my testing scenario. 
What do you purpose? Uncomment or delete these lines? Because rocket.chat (my example) did not have these lines.

Reply:
@cakrit @Ferroin you know more about this script than I do. Please help here.

Reply:
Uncomment them, they should be there.  Most of the mechanisms that don't have them do odd things that don't work with them, but Hangouts should definitely work correctly with multiple roles based on what I can tell from the code being added.

Reply:
@Ferroin @paulfantom Done.